canonical-websites/docs.ubuntu.com
# Update the styling of the search results page

## Done
Upgraded to vanilla-framework@1.6.7
Organised the Sass a little
Renamed homepage.scss to main.scss
Updated the report an issue link
Used the social icons in Vanilla
Added Juju option to the search results pages
Changed the snapcraft logo for the dark one

## QA
- Go to http://0.0.0.0:8007/
- Check the page looks the same as the live version
- Go to http://0.0.0.0:8007/search?q=cloud&domain=docs.jujucharms.com
- Check that the results looks good although they dont load
- Go to http://0.0.0.0:8007/search?q=cloud&domain=docs.ubuntu.com/docs
- Check the changes are expected

## Issue / Card
Fixes https://github.com/canonical-websites/docs.ubuntu.com/issues/150
Fixes https://github.com/canonical-websites/docs.ubuntu.com/issues/148
Fixes https://github.com/canonical-websites/docs.ubuntu.com/issues/147

## Screenshots
![screenshot-2018-4-18 ubuntu documentation](https://user-images.githubusercontent.com/1413534/38906583-7431ff74-42af-11e8-91a5-53e8813eee6e.png)
